py_trees/common.py

```diff
@@ -94,6 +109,17 @@ def __init__(self, children, synchronise=True):
             super().__init__(synchronise=synchronise)
             self.children = children
 
+        def __str__(self) -> str:
+            """
+            Human readable description.
+            """
+            description = "--" + self.__class__.__name__ + "("
+            description += u'\u26A1' if self.synchronise else "-"  # lightning bolt
```

Comment:
BTW although I like lightning bolts, I was wondering if you would like to consider this instead '🔁': https://en.wiktionary.org/wiki/%F0%9F%94%81